### ISSUE TITLE
--csv flag

### ISSUE THREAD

**Issue by doughsay** [unnawut/licensir#14]
Would be super awesome if there was a flag for exporting to CSV format.

Thanks for the tool!

**Comment by unnawut** [comment]
Great idea! Could you make an example of what the output should look like? And also how you would use it after that?

Asking this because there would be differences between having a header row or not, wrapping string in quotes (I don't think we need to escape anything, but who knows), etc., depending on how it'll be consumed.

Thanks again for the idea!

**Comment by doughsay** [comment]
Sure,

I used this tool and then had to manually edit the output to match this:

```
"absinthe", "1.4.16", "MIT"
"absinthe_phoenix", "1.4.4", "MIT"
"absinthe_plug", "1.4.7", "MIT"
"accept", "0.3.5", "MIT"
"amqp", "1.2.1", "MIT"
"amqp_client", "3.7.15", "MPL 1.1"
"apollo_tracing", "0.4.1", "MIT"
"appsignal", "1.10.9", "MIT"
"base64url", "0.0.1", "MIT"
"basic_auth", "2.2.4", "MIT"
...
```

From there I can import this into Google sheets or any other spreadsheet software easily.  I don't really care one way or the other if headers are included.  I did need the quotes, because sometimes the version string has commas in it, like `"Unsure (found: BSD, MIT)"`.

Also, I have one other suggestion (let me know if you'd rather I open a separate issue).  Umbrella applications will just perform the task for each sub-app, which may or may not be desired.  I needed the list of dependencies for my entire umbrella application, so I had to de-duplicate a lot of lines from my output. It would be great if there was a way to specify if you want each app separately or the entire umbrella's dependencies together.
